client/src/components/navigation/Navbar.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./Navbar.scss";
+import { AiOutlineHome } from "react-icons/ai";
+import { SlNotebook } from "react-icons/sl";
+import { TfiAnnouncement } from "react-icons/tfi";
+import { BiMessageAlt } from "react-icons/bi";
+import { TbSchool } from "react-icons/tb";
+import { CgProfile } from "react-icons/cg";
+
+import { Link } from "react-router-dom";
```

Comment:
I will be moving the static content(images and logos) inside the public folder.. as we need to serve those to use by the server (backend)!